lwgenerate/azure/azure.go

```diff
@@ -60,12 +66,24 @@ type GenerateAzureTfConfigurationArgs struct {
 	StorageLocation string
 
 	LaceworkProfile string
+
+	// Existing Event Hub Namespace
+	ExistingEventHubNamespace bool
+
+	// Event Hub Namespace Name
+	EventHubNamespaceName string
+
+	// Azure region where the event hub for logging will reside
+	EventHubLocation string
+
+	// Number of partitions in the Event Hub for logging
+	EventHubPartitionCount int
 }
 
 // Ensure all combinations of inputs are valid for supported spec
 func (args *GenerateAzureTfConfigurationArgs) validate() error {
 	// Validate one of config or activity log was enabled; otherwise error out
-	if !args.ActivityLog && !args.Config {
+	if !args.ActivityLog && !args.Config && !args.EntraIdActivityLog {
 		return errors.New("audit log or config integration must be enabled")
```

Comment:
Add entra to this list

Reply:
Entra is still audit log so I didn't change the error message, but I did add it to the if-condition